locallibs/install.py

```diff
@@ -87,6 +88,18 @@ def install_extras(framework_path, version="2.7", requirements_file=None,
                    install_wheel=False, upgrade_pip=False):
     """install all extra pkgs into Python framework path"""
     print()
+    python_guard_path = expanduser("~/Library/Python/%s") % version
+    if os.path.exists(python_guard_path):
+        print('*********************************************************')
+        print('*** Python user files exist that conflict with the    ***')
+        print('*** version of relocatable python you are trying to   ***')
+        print('*** create. This will likely result in site-packages  ***')
+        print('*** that are not installed or out-of-date. Please     ***')
+        print('*** remove these files or create this package on a    ***')
+        print('*** clean computer. The files are located at:         ***')
```

Comment:
replacing "on a clean computer" with "under a fresh user account" might be more helpful.